src/H5Gdeprec.c

```diff
@@ -719,11 +719,10 @@ H5Gset_comment(hid_t loc_id, const char *name, const char *comment)
  *
  * Note:	Deprecated in favor of H5Oget_comment/H5Oget_comment_by_name
  *
- * Return:	Success:	Number of characters in the comment counting
- *				the null terminator.  The value returned may
- *				be larger than the BUFSIZE argument.
+ * Return:	Success: Number of characters in the comment.  The value
```

Comment:
This should probably go the other way as in "Number of characters in the comment, excluding the NUL terminator character."

Reply:
@jhendersonHDF I missed this, I'll put it in one of my next PRs.